client/ayon_maya/plugins/publish/validate_skeleton_top_group_hierarchy.py

```diff
@@ -25,18 +37,12 @@ def process(self, instance):
         if not self.is_active(instance.data):
             return
 
-        invalid = []
-        skeleton_mesh_data = instance.data("skeleton_mesh", [])
-        if skeleton_mesh_data:
-            invalid = self.get_top_hierarchy(skeleton_mesh_data)
-            if invalid:
-                raise PublishValidationError(
-                    "The skeletonMesh_SET includes the object which "
-                    "is not at the top hierarchy: {}".format(invalid))
-
-    def get_top_hierarchy(self, targets):
-        targets = cmds.ls(targets, long=True)  # ensure long names
-        non_top_hierarchy_list = [
-            target for target in targets if target.count("|") > 2
-        ]
-        return non_top_hierarchy_list
+        skeleton_mesh_nodes = instance.data("skeleton_mesh", [])
+        if not skeleton_mesh_nodes:
+            return
```

Comment:
So - it _is_ still valid to have no `skeleton_mesh` set. Correct?

Reply:
Please confirm :)

Reply:
Oh Github - sometimes you're nice, sometimes you're JUST cropping off the details that I should've been reading from the code to begin with. This is NOT about animation - but about the `fbx.rig` instance.

Also - the code IS actually similar to before.
And, worse I think, I even suggested this code change at some point in this PR :')

Sorry for wasting your time - resolving.